providers/docker/src/airflow/providers/docker/operators/docker.py

```diff
@@ -291,6 +298,8 @@ def __init__(
         self.tmp_dir = tmp_dir
         self.user = user
         self.mounts = mounts or []
+        for mount in self.mounts:
+            mount.template_fields = ("Source", "Target", "Type")
```

Comment:
Oh wow... I had no idea you can do that :)

Reply:
There are tests to back it up